lib/logger.ts

```
@@ -0,0 +1,106 @@
+// lib/logger.ts
+
+type LogLevel = "debug" | "info" | "warn" | "error";
+
+/**
+ * Structured logger for consistent log formatting
+ */
+export const logger = {
+  debug: (message: string, data?: any) => log("debug", message, data),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)